import-io/python-lib/importio_utils.py

```diff
@@ -54,21 +59,24 @@ def run(build_query):
             response = requests.get(request_url)
             json = response.json()
         except Exception as e:
-            print 'request to import.io failed'
-            print e
-            print 'response was:\n',response
-            raise
+            logger.error('request to import.io failed')
+            logger.error(e)
+            logger.error('response was: {}'.format(response))
+            raise ValueError
```

Comment:
aren't we masking exceptions? Why not just throw the actual exception? (There's sometimes reason to do this but I don't know if that is the case here as a number of things could be going wrong and it might help user to know)